checks/phpcs.php

```diff
@@ -110,16 +110,39 @@ protected function phpcs_result_to_warnings( $result ) {
 				}
 
 				$source_code = esc_html( trim( file( $this->path . '/' . $filename )[ $message['line'] - 1 ] ) );
+				$plugin_data = get_plugins( '/' . $this->slug );
+				$edit_link   = '';
+
+				$edit_link = sprintf(
+					'<a href="%1$s" title="%2$s" aria-label="%2$s" target="_blank">%3$s</a>',
+					esc_url(
+						add_query_arg(
+							[
+								'plugin' => rawurlencode( $this->slug . '/' . array_key_first( $plugin_data ) ),
+								'file'   => rawurlencode( $this->slug . '/' . $filename ),
+								'line'   => rawurlencode( $message['line'] ),
+							],
+							admin_url( 'plugin-editor.php' )
+						)
+					),
+					sprintf(
+						/* translators: %1$s is the path to a plugin file. */
+						esc_attr__( 'View %1$s in the plugin file editor.', 'plugin-check' ),
```

Comment:
```suggestion
						/* translators: %s is the path to a plugin file. */
						esc_attr__( 'View %s in the plugin file editor.', 'plugin-check' ),
```

For single translation variable use `%s`